documentation/modules/snippet_ova_tech_preview.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: SNIPPET
+
+Migration using Open Virtual Appliance (OVA) files as source providers is a Technology Preview.
```

Comment:
@RichardHoch 
Why are OVA files 'source providers', in plural? Maybe 'as a source provider' would be better? Elsewhere, you wrote "If your source provider is a set of one or more OVA files...".

Files being a 'source provider' also sounds strange, but it seems to be unavoidable here.
Also, isn't this line redundant? It says exactly what the note that follows says,